### ISSUE TITLE
Provide OOTB Connectors as part of a docker-compose for Self-Managed

### ISSUE THREAD

**Issue by tmetzke** [camunda/connector-sdk#75]
__Is your feature request related to a problem? Please describe.__

We enable Self-Managed (SM) users to run a [docker-compose file](https://github.com/camunda/camunda-platform/tree/main) to spin up Zeebe, Operate, and Tasklist as a getting started experience. We don't have this for Connectors in SM.

__Describe the solution you'd like__

Either of the following options:

- Integrate the Connector Job Worker Runtime and all OOTB Connectors in the existing Get Started docker-compose.
- Create a separate docker-compose that enables running the Connector Job Worker Runtime and all OOTB Connectors.

__Describe alternatives you've considered__

Provide manual downloads to the JARs for the Connector Job Worker Runtime and all OOTB Connectors and a description of how to run them all together. 

**Note**: We don't have a place yet to offer the JARs for download. Every Connector has its own release cycle, we cannot ship a set of OOTB Connectors with an SDK release, for example.

__Additional context__

n/a

**Comment by tmetzke** [comment]
@PreetiNirwal, this is the ticket for SM Trial users and docker-compose.
@menski, we'd need your timely input on this regarding
* What would we need in order to make this happen? One docker image that contains the Job Worker Runtime and all OOTB Connectors? One docker image per OOTB Connector? What would those single images contain? 
* Do we see this possible until September 27th, 2022?

**Comment by menski** [comment]
@tmetzke 

> What would we need in order to make this happen? One docker image that contains the Job Worker Runtime and all OOTB Connectors? One docker image per OOTB Connector? What would those single images contain?

What is our goal, as far as I understand it's about enabling users to try out connectors, I assume we don't aim that users can explicitly pick which connectors they want to try out. So having a fat ootb connector image which contains a worker runtime with all available connectors would solve that correct? For that we would only have to publish a docker image which we can build and bundle everything together, we don't have to publish jar files. But does our job worker runtime already supports multiple connectors?

> Do we see this possible until September 27th, 2022?
If we come to an agreement fast and someone can focus on it yes, otherwise it's a bit short notice.

**Comment by tmetzke** [comment]
> What is our goal, as far as I understand it's about enabling users to try out connectors, I assume we don't aim that users can explicitly pick which connectors they want to try out. 

@PreetiNirwal, correct me if I'm wrong, but this is mainly for the getting started experience, right?

> So having a fat ootb connector image which contains a worker runtime with all available connectors would solve that correct? For that we would only have to publish a docker image which we can build and bundle everything together

That should suffice, yes.

> But does our job worker runtime already supports multiple connectors?

Yes, you can currently configure all the Connectors that should be available via ENV vars, see [documentation](https://docs.camunda.io/docs/components/integration-framework/connectors/custom-built-connectors/connector-sdk/#pre-packaged-runtime-environment).

**Comment by tmetzke** [comment]
> we don't have to publish jar files

That is true for this use case. However, to also enable the full Enterprise use case, we also need that anyways. I was just wondering if we can do this in the scope of this ticket here as well but we might as well create a new one for that.

**Comment by PreetiNirwal** [comment]
Priority 1: For Trial users, let's bundle all Connectors for them (eg. they can't pick & choose which Connectors they want to trial).
Priority 2: For Enterprise users, we would like them to select which Connectors they want in Production.

We want priority 1 for Sep 27. For priority 2, I am *inclined to say if it's not ready by Sep 27 it's ok and if it can be ready by ~mid October we should be ok.
